### ISSUE TITLE
Can't read config.js

### ISSUE THREAD

**Issue by screendriver** [systemjs/builder#315]
I can't bundle anymore.

```sh
[09:24:49] TypeError: Cannot read property 'toString' of undefined
    at Builder.executeConfigFile (/home/me/development/project/node_modules/systemjs-builder/lib/builder.js:162:22)
    at lib$rsvp$$internal$$tryCatch (/home/me/development/project/node_modules/systemjs-builder/node_modules/rsvp/dist/rsvp.js:493:16)
    at lib$rsvp$$internal$$invokeCallback (/home/me/development/project/node_modules/systemjs-builder/node_modules/rsvp/dist/rsvp.js:505:17)
    at lib$rsvp$$internal$$publish (/home/me/development/project/node_modules/systemjs-builder/node_modules/rsvp/dist/rsvp.js:476:11)
    at lib$rsvp$asap$$flush (/home/me/development/project/node_modules/systemjs-builder/node_modules/rsvp/dist/rsvp.js:1198:9)
    at doNTCallback0 (node.js:408:9)
    at process._tickCallback (node.js:337:13)

```

My `gulpfile.babel.js`

```js
import gulp from 'gulp';
import Builder from 'systemjs-builder';

gulp.task('bundle', () => {
  const builder = new Builder();
  return builder.loadConfig('./config.js')
    .then(() => builder.bundle('index.js', 'bundle.js'));
});
```

I looked at `builder.js:162:22` and it seems that `source` is undefined. My `./config.js` exists though. I can open it with the nodejs `fs` module within my `gulpfile.babel.js`.

I am using jspm `0.16.5` and systemjs-builder `0.14.1`.

**Comment by kelsmj** [comment]
I am getting the same thing after upgrading to the latest.

**Comment by guybedford** [comment]
Thanks I've added a fix, will release a patch shortly.

**Comment by guybedford** [comment]
Released in 0.14.2.
